language/move-prover/bytecode-to-boogie/src/prelude.bpl

```diff
@@ -95,7 +99,7 @@ function {:inline 1} AddValueArray(a: ValueArray, v: Value): ValueArray {
 }
 
 function {:inline 1} RemoveValueArray(a: ValueArray): ValueArray {
-    ValueArray(v#ValueArray(a)[l#ValueArray(a) := DefaultValue], l#ValueArray(a) - 1)
+    ValueArray(v#ValueArray(a)[l#ValueArray(a)-1 := DefaultValue], l#ValueArray(a) - 1)
```

Comment:
nit: please add spaces before and after `-`

Reply:
Done.